concourse/Dockerfile

```diff
@@ -9,9 +9,9 @@ RUN echo "deb https://dl.yarnpkg.com/debian/ stable main" | tee /etc/apt/sources
     && apt update -y \
     && apt install -y yarn
 
-RUN wget https://dl.google.com/linux/direct/google-chrome-stable_current_amd64.deb \
-    && apt-get install -y --no-install-recommends ./google-chrome-stable_current_amd64.deb \
-    && rm ./google-chrome-stable_current_amd64.deb
+RUN wget https://dl.google.com/linux/chrome/deb/pool/main/g/google-chrome-stable/google-chrome-stable_88.0.4324.182-1_amd64.deb \
```

Comment:
Great commit message nice and clear!

One thing i'm wondering here, I know @danacotoran said the google past versions go back a way, do the host versions going all the way back? 

Wondering if chrome might hit version 98 (ten more) or six months goes by and we find this stable version 404-ing from their servers?

I guess at least then the stack trace would be a lot clearer!

Reply:
I sincerely hope we won't use a version of Chrome (or any software really) that's more than (ideally) a couple of versions behind, surely not 10! 😄

In any case, if they do remove old versions, we could use that as a "reminder" to update - it won't go unnoticed for sure.

Reply:
But joking aside, I think the main takeaway here is that we should treat Chrome like any other dependency and pin it to a specific version. For that, we need to think about a process to update it (that should be ideally automated, similar to what dependabot does for our other dependencies).